Integrations/integration-trendMicroDsm.yml

```diff
@@ -0,0 +1,327 @@
+description: Trend Micro Deep Security
```

Comment:
I would check with dan if this is the description he wants (usually it's uses trend micro...)